rest-api-spec/src/main/resources/rest-api-spec/test/cluster.state/10_basic.yml

```diff
@@ -18,3 +18,18 @@
   - is_true: master_node
   - gte: { compressed_size_in_bytes: 50 }
   - is_true: compressed_size
+
+---
+"get cluster state returns cluster_uuid at the top level":
+  - skip:
+      version:  " - 6.99.99"
```

Comment:
There will be a follow-up PR to change this to `6.3.99` after this PR is merged / backported to 6.x (6.4).

Reply:
https://github.com/elastic/elasticsearch/pull/30264 as promised